code/testapp/build.gradle

```diff
@@ -47,24 +47,8 @@ dependencies {
 
     implementation project(":analytics")
 
-    implementation 'com.adobe.marketing.mobile:core:2.0.0-SNAPSHOT'
-//    implementation('com.adobe.marketing.mobile:analytics:2.0.0-SNAPSHOT'){
-//        transitive = false
-//    }
-    implementation('com.github.adobe.aepsdk-core-android:lifecycle:dev-v2.0.0-SNAPSHOT'){
-        transitive = false
-    }
-    implementation('com.github.adobe.aepsdk-core-android:identity:dev-v2.0.0-SNAPSHOT'){
-        transitive = false
-    }
-    implementation('com.github.adobe:aepsdk-assurance-android:2.0.0-alpha.1'){
-        transitive = false
-    }
-
-//    implementation 'com.adobe.marketing.mobile:lifecycle:1.+'
```

Comment:
I have to keep them for now since we need to do 1.x -> 2.0 upgrade testing. Can we remove it later?

Reply:
+1, let's make those changes locally.

Reply:
@yangyansong-adbe said we can remove this now. He is done with the testing.